### ISSUE TITLE
[Bug] {{input type="checkbox"}} `change` no longer clobbers change in 3.27+

### ISSUE THREAD

**Issue by runspired** [emberjs/ember.js#19763]
### 🐞 Describe the Bug

This one is a bit sad-face all around, so apologies to everyone in advance that I'm asking for a revert to an annoyingly bad behavior, albeit one that worked as "public" API for several major versions of Ember.

As part of the [built-in component modernization](https://github.com/emberjs/rfcs/blob/master/text/0671-modernize-built-in-components-1.md) it appears that the two-way binding of the `checked` value is no longer handled in the same override-able way. [See here](https://github.com/emberjs/ember.js/pull/19382/files#diff-0122bfc3d95c56a846a65cd41d525c229d803b59af7e197633b64e8ace9e9121R247)

Previously, the following would result in the value of `checked` effectively being one-way with the update handled by whatever action the user supplied to `change`. For instance:

```hbs
{{input type="checkbox" checked={{isItemSelected}} change=(action "changeValueAction" itemBeingSelected)}}
```

However beginning in 3.27 this will instead mutate `isItemSelected` *and* fire the change action, leading the value to be mutated twice (and thus in most cases leading to it toggling right back to where it began).

**Comment by runspired** [comment]
If anyone else hits this, I have a codemod that will fix it for you.

**Comment by johanrd** [comment]
@runspired yes, definitely a foot gun here. Though, considering [RFC 671](https://rfcs.emberjs.com/id/0671-modernize-built-in-components-1), this would require an RFC to revert? Maybe a better future is to deprecate the `Input` and `Textarea` components altogether? ([Last attempt was closed](https://github.com/emberjs/rfcs/issues/498), but maybe we are ready for it now?)

On the good side, [the API docs](https://api.emberjs.com/ember/release/classes/Ember.Templates.components/methods/Input?anchor=Input) at least mention the native `input` element for one way binding:
> If you wanted a one-way binding, use `<input type="checkbox">` with `checked` and the `input` event instead.

**Comment by NullVoxPopuli** [comment]
I'm going to cloes this because if you use angle-bracket syntax two-way binding works

https://limber.glimdown.com/edit?c=JYWwDg9gTgLgBAbzgSQHZgK7wL5wGZQQhwDkAAgKYgBGFUA9AMZGSoWowkDcAUKJLERwYUAIaMA1hQAmcXASKkyAcwA2oEHXojxE4KmXcePZqgDO8YGYDCACwqSZcALzCxj6QAo8o1WYoAlLw8ADwwVGCqouEAfDxwcCFRtKoxVnDA8Iz2HgD88QmJaJjwZDAAnmAUzgBE2Q4S1BAAHjVwZPUezggIVnYNMgB0AG6%2BGBTYuPRxCSH0yRSpxrNgUBQxPQBSAMoA8gBygxZQ%2BsrAeOVwnrvUAFYOljY5UtLOfc9Do6rjAZNzq%2BtQtoIlFYkA&format=gjs


```gjs
import { Input } from '@ember/component';
import { tracked } from '@glimmer/tracking';

const isChecked = tracked(false);

<template>
  <label>is it checked?
    <Input @type="checkbox" @checked={{isChecked.value}} />
  </label>

  <pre>{{JSON.stringify (Object isChecked=isChecked.value)}}</pre>
</template>
```

Tested with 7.3.0-alpha


> [!NOTE]
> `action` also no longer exists
